api/src/main/java/com/cloud/resource/ResourceState.java

```diff
@@ -16,12 +16,12 @@
 // under the License.
 package com.cloud.resource;
 
+import com.cloud.utils.fsm.StateMachine;
+
 import java.util.Arrays;
 import java.util.List;
 import java.util.Set;
 
-import com.cloud.utils.fsm.StateMachine;
-
```

Comment:
nit- Unintended changes - in case we care about commit cleanliness (skip if not).

Reply:
I think thats happened during the merge. My IntelliJ sorts the imports differently, for whatever reason..

Reply:
Yeah, happens with most of us. In case not done already, you can perhaps use the code style guideline from ACS page to configure them to community defined guidelines https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions (IntelliJ Idea section)